src/groovy/org/transmartproject/db/dataquery/highdim/snp_lz/SnpLzModule.groovy

```diff
@@ -171,7 +175,11 @@ class SnpLzModule extends AbstractHighDimensionDataTypeModule {
                 property 'snp.trialName',              'trialName'
 
```

Comment:
The geneInfo? It's possible, but in the documentation the client gave us they describe snp_id (the one that links to `de_snp_info`) as a legacy column, so I avoided using it.